doc/build-helpers/fetchers.chapter.md

```diff
@@ -519,15 +519,80 @@ See [](#chap-pkgs-fetchers-caveats) for more details on how to work with the `ha
 
 ## `fetchzip` {#sec-pkgs-fetchers-fetchzip}
 
-Downloads content from a given URL (which is assumed to be an archive), and decompresses the archive for you, making files and directories directly accessible.
+Returns a [fixed-output derivation](https://nixos.org/manual/nix/stable/glossary.html#gloss-fixed-output-derivation) which downloads content from a given URL (assumed to be an archive), and decompresses the content for you, making files and directories directly accessible.
 `fetchzip` can only be used with archives.
```

Comment:
Would be interesting to know what kind of archives are supported:

- https://github.com/nixos/nixpkgs/blob/master/pkgs/stdenv/generic/setup.sh#L1040

just from looking I'd say:

*.tar.xz, *.tar.lzma, *.txz, *.tar, *.tar.*, *.tgz, *.tbz2 and *.tbz

So for me it looks like *.zip file is actually not supported by default. (Unlike the name suggests)

You'd need to pass `unpackCmd` with an apropriate `tar` invocation